lib/google.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+create_image_from_snapshot(){
+    IMAGE=$1
+    FAMILY=$2
+    DESCRIPTION=$3
+    SNAPSHOT=$4
+    gcloud compute images create "$IMAGE" --family "$FAMILY" --description "$DESCRIPTION" --source-snapshot "$SNAPSHOT"
+}
+
+create_image_from_bucket(){
+    IMAGE=$1
+    FAMILY=$2
+    DESCRIPTION=$3
+    IMAGE_FILE=$4
+    gcloud compute images create "$IMAGE" --family "$FAMILY" --description "$DESCRIPTION" --source-uri "gs://$BUCKET_NAME/$IMAGE_FILE"
+}
+
+delete_image(){
+    IMAGE=$1
+    image_name=$(gcloud compute images list --project $GCE_PROJECT --filter "family = $FAMILY AND name = $IMAGE" --format json | jq -r '.[]|.name')
+
+    if ! [ -z $image_name]; then
```

Comment:
Perhaps missing space?

Reply:
Let's shellcheck the whole thing to make sure it's not broken in any way.

Reply:
or just `if [ -n "$image_name" ]; then...`